docs/book/src/introduction.md

```diff
@@ -1,8 +1,7 @@
 # Kubernetes Cluster API Provider IBM Cloud
 
 <p align="center">
-<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  width="100x"> 
-<img src="../images/ibm-cloud.svg" height="100x" width="200x" alt="IBM Cloud">
```

Comment:
can we delete this above `../images/ibm-cloud.svg` if not used anymore?

Reply:
Done.